Week3/homework/js-exercises/2-dog-ages.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+
+function calculateDogAge(humanYears){
+if (humanYears > 0 ){
+const dogYears = humanYears * 7;
+return "Your doggie is " +dogYears+ " years old in dog years!"
+}else return "Your dog's age must be a positive value";
```

Comment:
Ah. The logic seems alright but there is an indentation problem which makes it harder to read. Please make sure that the code is well formatted as it is also an important aspect of development. 🙂 

Also, the homework seems to have a type. The condition to convert human years to dog years should be `humanYears / 7` instead of `*` 😛

Reply:
I think I didn't get the logic right. As far as I understood, if a dog is 3 years old regarding our time concepts, it is equal to 21 years old humans life span in dog year, so, for example, he/she would be a young adult

Reply:
Exactly. If you try to put it in your logic currently which is **dogYears = humanYears * 7**, it doesn't fit well, right?

Currently, dogYears(3) is equal to humanYears(21) * 7 which isn't correct. But dogYears(3) should be equal to humanYears(21) / 7.